parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -102,10 +98,10 @@ public static WriterVersion fromString(String name) {
   private final int columnIndexTruncateLength;
   private final int statisticsTruncateLength;
 
-  // The key-value pair represents the column name and its expected distinct number of values in a row group.
-  private final Map<String, Long> bloomFilterExpectedDistinctNumbers;
+  // The expected NDV (number of distinct values) for each columns
+  private final ColumnProperty<Long> bloomFilterNDVs;
```

Comment:
Is it max value?

Reply:
`ColumnProperty` is created to handle different values per column names. It may contain one value for each column.